### ISSUE TITLE
For long hover text, content is cut and not useful

### ISSUE THREAD

**Issue by lynlevenick** [joaotavora/eglot#117]
I'm using solargraph `0.28.1` and eglot at the commit hash `f482c3ecf766a9230ea46a5f54b2bd7821c11233`.

I wrote a file, `test.rb` with the content:

```ruby
StandardError
```

Moving the point over `StandardError` results in the following display:

![image](https://user-images.githubusercontent.com/346549/45804939-5ac5b480-bc72-11e8-95b2-66d4d0b57802.png)

Solargraph appears to return the full documentation:

```
$ ri StandardError
= StandardError < Exception

(from ruby core)
------------------------------------------------------------------------------
The most standard error types are subclasses of StandardError. A rescue clause
without an explicit Exception class will rescue all StandardErrors (and only
those).

  def foo
    raise "Oups"
  end
  foo rescue "Hello"   #=> "Hello"

On the other hand:

  require 'does/not/exist' rescue "Hi"

raises the exception:

  LoadError: no such file to load -- does/not/exist
------------------------------------------------------------------------------
(END)
```

I'm not sure that there's a great solution to making this particular instance work well, but it seems like there could be better handling of this case to indicate that the full content is not being displayed, and perhaps to favor the beginning of it rather than the end.

**Comment by fbergroth** [comment]
#111 attempts to deal with this: hovering only shows the first line, and to view the full documentation you can use `M-x eglot-help-at-point`.

**Comment by joaotavora** [comment]
I'm aware of the problem, but unfortunately don't have time to analyze fully. I don't remember the bc reason for pruning the last part the string anymore, and this is why I'm weary of merging #111

**Comment by muffinmad** [comment]
@lynlevenick Looks like this issue is fixed in recent eglot version. Thanks for the patience! ;-)

**Comment by joaotavora** [comment]
Yep this seems to be fixed by #459
